packages/react-testing/README.md

```diff
@@ -294,6 +295,34 @@ myComponent.triggerKeypath('action.onAction');
 expect(spy).toHaveBeenCalled();
 ```
 
+### <a name="debug"></a> `debug(): string`
+
+Returns a text representation of either the root node, or any element within the mounted graph. `debug()` has two forms, `debug(verbosity?: number = 1, depth?: number)` for simple usage and `debug({verbosity?: number, depth?: number})` for more direct output control. `verbosity` defines the level of expansion that non-scalar props experience, the default value of `1` will expand objects for example one level deep. `depth` defines the number of children printed, by default all children are printed. Typical usage should not require any parameters as the default `verbosity` and `depth` should be appropriate for the majority of inspections.
+
+```tsx
+function ObjectText({data}: {data: {}}) {
+  return <span>{JSON.stringify(data)}</span>;
+}
+
+function MyComponent({onClick}: {onClick(id: string): void}) {
+  return (
+    <div>
+      <button type="button" onClick={() => onClick(String(Math.random()))}>
+        <ObjectText data={{a: {very: {deep: {data: {object: 'with text'}}}}}} />
+      </button>
+    </div>
+  );
+}
+
+const wrapper = mount(<MyComponent />);
+// return the whole structure with one level of prop verbosity
+wrapper.debug();
+// return only the div and button without any other children
+wrapper.find(div)!.debug({depth: 1});
+// find by name and return all children with verbose props details
+wrapper.findWhere(type => type && type.name === 'button')!.debug(9);
```

Comment:
These examples are kind of weird because they just produce strings but don’t log them or anything :/

Reply:
well i guess the intent is that you run this in the debug execution console, so the result is automatically printed. i could wrap with `console.log` for more clarity